dbms/src/TestUtils/InputStreamTestUtils.cpp

```diff
@@ -273,10 +273,8 @@ ::testing::AssertionResult InputStreamVSBlockUnrestrictlyCompareColumns(
     const Strings & colnames,
     const ColumnsWithTypeAndName & columns)
 {
-    RUNTIME_CHECK(stream != nullptr, Exception(fmt::format("The first param of ASSERT_INPUTSTREAM_COLS_UR, `{}` is nullptr!", stream_expr)));
-    RUNTIME_CHECK(
-        colnames.size() == columns.size(),
-        Exception(fmt::format("The length of second and thrid param of ASSERT_INPUTSTREAM_COLS_UR not match! {} != {}", colnames.size(), columns.size())));
+    RUNTIME_CHECK(stream != nullptr, stream_expr);
```

Comment:
```suggestion
    RUNTIME_CHECK_MSG(stream != nullptr, "The first param of ASSERT_INPUTSTREAM_COLS_UR, `{}` is nullptr!", stream_expr);
```
It seems that exception.message() is changed too much

Reply:
The current produced exception message is:

```
Check stream != nullptr failed, stream_expr = <expr>
```

Does it looks good to you that provided sufficient information? Or do you think this would be better?

```
Check stream != nullptr failed: The first param of ASSERT_INPUTSTREAM_COLS_UR, `<expr>` is nullptr!"
```

Reply:
Thanks for pointing out! I have updated these messages in the macro invocation to be more friendly for macro users. PTAL again, thanks!